README.md

```diff
@@ -22,6 +22,10 @@ When instantiating the plugin, your `opts` need the correct fields.
   "account": "can be anything; only used for logging",
   "token": "channel name in MQTT server",
   "limit": "amount of negative balance that can exist",
+  "warnLimit": "amount of negative balance before warning to settle",
```

Comment:
Not sure I like the term `warn`. Warning to me is an exceptional condition, not something that happens all the time normally. Maybe `highSettlementTrigger`/`highTrigger`?